apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts

```diff
@@ -1,9 +1,10 @@
 import { AggregatedReadDTO } from '@energyweb/origin-drec-api-client';
 import { useTheme } from '@material-ui/core';
+import { getMegawattsFromWatts } from 'utils';
 
 export const useGenerateChartDataset = (reads: AggregatedReadDTO[]) => {
     const theme = useTheme();
-    const currentData = reads.map((read) => read.value);
+    const currentData = reads.map((read) => getMegawattsFromWatts(read.value));
```

Comment:
We have a `PowerFormatter` and `EnergyFormatter` in `@energyweb/origin-ui-utils` package for the same formatting

Reply:
From what I know PowerFormatter just formats the data as a presentation layer and returns it as a string. PowerFormatter from @energyweb/origin-ui-utils had some issues so I used an internal one. I can check EnergyFormatter and if it makes sense, I will replace this method.

Reply:
I checked both methods but they return different values than what I need, and as string.